### DISCUSSION TITLE
Banner

### DISCUSSION THREAD

**Discussion by murrlipp** [ministryofjustice/moj-frontend#835] [RELEASED: Components, pages and patterns]
The [banner component](https://design-patterns.service.justice.gov.uk/components/banner/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by robertjmccarthy** [comment]
@chrispymm commented on #619 that _the GOV.UK notification banner allows for the role to be set to "alert" and has JS to autofocus the banner on page load in this case. It would be helpful if our component had the same behaviour, as it is beneficial for accessibility._

#619 will be closed as this discussion will be reviewed when the banner component is improved. The banner component has been added to the design system team's roadmap for priority improvement.
